### DISCUSSION TITLE
Render config

### DISCUSSION THREAD

**Discussion by andershagman** [netbox-community/netbox#13231] [Help Wanted!] [answered]
Netbox version: 3.5.4

I´m testing the new config render thing and am stuck with how to get all interfaces from Virtual Chassis.
```
{% for interface in device.interfaces.all() %}
```
only gives you the master interfaces.

**Comment by candlerb** [comment, marked as answer]
Try `vc_interfaces` instead of `interfaces`

**Reply by andershagman** [reply]
Tanks, as usual a correct answer.

As a tip for me and others. Look at the models in nbshell. Command completion works.

```python
>>> d = Device.objects.get(name="HMA-201-l2-a sw1")
>>> d.interfaces.all().count()
50
>>> d.vc_interfaces().count()
98
```
